sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -208,10 +223,22 @@ private static bool AddStringsToList(BinaryExpressionSyntax node, List<LiteralEx
                 SqlStartQueryKeywords.Any(s => firstString.StartsWith(s, StringComparison.OrdinalIgnoreCase));
 
             /**
-             * The '@' symbol is used for named parameters.
+             * The '@' symbol is used for named parameters. The '{' and '}' symbols are used in string interpolations.
              * We ignore other non-alphanumeric characters (e.g. '>','=') to avoid false positives.
              */
-            private static bool IsAlphaNumericOrAt(char c) => char.IsLetterOrDigit(c) || c == '@';
+            private static bool IsAlphaNumericOrAt(char c) => char.IsLetterOrDigit(c) || c == '@' || c == '{' || c == '}';
+        }
+
+        internal class StringWrapper
```

Comment:
Any reason not to make this class private?

Reply:
No, indeed. Updated class to private